codec-http2/src/main/java/io/netty/handler/codec/http2/Http2ConnectionHandler.java

```diff
@@ -167,7 +167,7 @@ public void onHttpServerUpgrade(Http2Settings settings) throws Http2Exception {
     @Override
     public void flush(ChannelHandlerContext ctx) throws Http2Exception {
         // Trigger pending writes in the remote flow controller.
-        connection().remote().flowController().writePendingBytes();
+        encoder.flowController().writePendingBytes();
```

Comment:
@nmittler +1. It should be the same thing, however this is shorter, and consistent with accesses to the flow controller in this class.